galleries/users_explain/artists/imshow_extent.py

```diff
@@ -3,8 +3,8 @@
 
 .. _imshow_extent:
 
-*origin* and *extent* in `~.Axes.imshow`
-========================================
+The *origin* and *extent* in `~.Axes.imshow`
+============================================
```

Comment:
I wonder if `origin` and `extent` are really important to place in the title here; maybe it should have been something like "Positioning and orientation of `~.Axes.imshow` images"